### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nikshepard**
1. Some updates for already completed exercises - detailed comments on the lesson and improved grammar.
2. Added empty forms for the following lessons.

### CODE REVIEW

File: 3-Lines/Lesson-15.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// ' '
+// ''
+// Любой одиночный символ в кавычках — это строка. Пустая строка '' — это тоже строка.
+
+// Стандарт кодирования airbnb, рекомендует использовать, по возможности, одинарные кавычки.
+console.log('Dracarys!');
+
+// Если мы хотим написать кавычки, то необходимо воспользоваться экраном - \
+// Экранируется только ", так как в этой ситуации двойные кавычки имеют специальное значение
+console.log("Dragon's mother said \"No\"");
```
Comment (konrad-molitor) [inline 3-Lines/Lesson-15.js:15]: FYI: Modern linter configs prefer ' over " and will complain about string literals enclosed with "". 
FYI2: If we use different quotes, there is no need in backslash
```javascript
console.log('She said "No"');
console.log("She said 'No'");
```

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
-# Learn-JS
```
Comment (konrad-molitor) [inline README.md:1]: There is an example why you always should add a new line to the end of file)

File: 1-Basics/Lesson-3.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Lesson-3 Statements
+
+// Можно сделать так
+console.log('Mother of Dragons.');
+console.log('Dracarys!');
+
+// Но так будет более правильно и красиво
+// \n переносит значение на следующую строку
+console.log('\nRobert\nStannis\nRenly')
```
Comment (konrad-molitor) [inline 1-Basics/Lesson-3.js:9]: There is another way to store (and print) multiline strings - just use backticks `string`;
Like this:
```javascript
const text = `Robert
              Stannis
              Renly`;
```

**Review by konrad-molitor — APPROVED** [review]

File: 3-Lines/Lesson-15.js (hunk 1 above)
Comment (nikshepard) [reply 3-Lines/Lesson-15.js:15]: I agree with the comments. 
In the specified example, console.log("Dragon's mother said \"No\"") also uses quotation marks ', which somehow leads to escaping either 'or', which is the purpose of the task.

**Pull request merged**